packages/mjml-chart/src/index.js

```diff
@@ -0,0 +1,308 @@
+    "examples": [
+      "b0,lg,0,f44336,0.3,03a9f4,0.8"
+    ],
+    "required": false
+  },
+  {
+    "name": "chan",
+    "pattern": "/^[0-9]{2,4}(?:,(?:easeInQuad|easeOutQuad|easeInOutQuad|easeInCubic|easeOutCubic|easeInOutCubic|easeInQuart|easeOutQuart|easeInOutQuart|easeInQuint|easeOutQuint|easeInOutQuint|easeInSine|easeOutSine|easeInOutSine|easeInExpo|easeOutExpo|easeInOutExpo|easeInCirc|easeOutCirc|easeInOutCirc|easeInElastic|easeOutElastic|easeInOutElastic|easeInBack|easeOutBack|easeInOutBack|easeInBounce|easeOutBounce|easeInOutBounce))?$/",
+    "examples": [
+      "1200",
+      "1300|easeInOutSine"
+    ],
+    "required": false
+  },
+  {
+    "name": "chli",
+    "pattern": "/^(?:.*)$/i",
+    "examples": [
+      "95K€",
+      "45%"
+    ],
+    "required": false
+  },
+  {
+    "name": "icac",
+    "examples": [
+      "accountId"
+    ],
+    "required": false
+  },
+  {
+    "name": "ichm",
+    "examples": [
+      "0785cf22a0381c2e0239e27c126de4181f501d117c2c81745611e9db928b0376"
+    ],
+    "required": false
+  }
+]
+
+/**
+ * @type {Array[String]}
+ */
+const forbiddenAttributes = [
+  "src",
+  "width",
+  "height",
+  "href",
+  "target"
+]
+
+const endingTag = true
+const baseStyles = {
+  table: {
+    borderCollapse: 'collapse',
+    borderSpacing: '0px'
+  },
+  img: {
+    border: 'none',
+    borderRadius: '',
+    display: 'block',
+    outline: 'none',
+    textDecoration: 'none',
+    width: '100%'
+  }
+}
+
+const strToRegExp = memoize((regexp) => {
+  const [__, pattern, flags] = /\/(.*)\/(.*)/.exec(regexp); // eslint-disable-line no-unused-vars
+  return new RegExp(pattern, flags);
+})
+
+function buildURL (mjAttribute) {
+
+  function buildQuery () {
+    return imageChartsParameters.reduce((query, attr) => {
+      const val = mjAttribute(attr.name);
+
+      assert(!(attr.required && val === undefined), `${attr.name} is required. Examples values ${attr.examples.join(', ')}`);
+
+      if (val === undefined) {
+        return query;
+      }
+
+      if (attr.pattern) {
+        assert(strToRegExp(attr.pattern).test(val), `"${val}" is an invalid value for ${attr.name}. Examples: ${attr.examples.join(', ')}`);
+      }
+
+      if (attr.enum) {
+        assert(attr.enum.includes(val), `"${val}" is not an valid value for ${attr.name}. Valid values are: ${JSON.stringify(attr.enum)}`);
+      }
+
+      query[attr.name] = val;
+      return query;
+    }, {});
+  }
+
+  return `https://image-charts.com/chart?${qs.stringify(buildQuery())}`;
+}
+
+@MJMLElement
+class Chart extends Component {
+
+  render () {
+    const {
+      mjAttribute
+    } = this.props
+
+    const [width, height] = mjAttribute('chs').split('x').map(toNumber)
+
+    // ensure forbidden attributes were not specified
+    forEach(this.props.mjml.get('attributes').toObject(), (value, attr) => {
```

Comment:
Why some values are forbidden ? You can't use lodash's `omit` or a `filter` ? 
In MJML3, we already check for forbidden (aka non-valid) attributes, so I think this is useless ?

I'm not a big fan of component output some stuff